package.json

```diff
@@ -45,7 +45,9 @@
     "mocha": "5.0.0",
     "commander": "2.8.1",
     "abi-decoder": "1.2.0",
-    "wait-port": "0.2.2"
+    "wait-port": "0.2.2",
+    "web3": "1.0.0-beta.37",
+    "web3-eth-accounts": "1.0.0-beta.37"
```

Comment:
Do we need below two lines since web3 package was installed thorugh mosaic-tbd npm previously?
Is it not the case now?
```
 "web3": "1.0.0-beta.37",
    "web3-eth-accounts": "1.0.0-beta.37"
```

Reply:
Yeah. Per the mosaic team, they are now `peerDependencies` in mosaic.js and so need to be specified as `devDependencies` here.